reference/service/saml-okta.md

```diff
@@ -92,26 +90,23 @@ a user's identity.
 
 ![SAML Application Metadata](../../images/reference/service/saml-okta/okta-xml-descriptor.png)
 
-With the block of XML text in your clipboard, navigate to your SAML organization's page on the
-Pulumi Cloud Console. The exact URL depends on the name of the organization, but it will be
-something like:
-https://app.pulumi.com/robot-co/settings
+With the block of XML text in your clipboard, open to the Pulumi Cloud Console and navigate to your SAML organization. Click the
+**Settings** tab, and then select **SAML SSO**.
 
-On the organization setting's page, paste the IDP metadata descriptor into the bottom card
-titled "SAML SSO Settings". Then click the "SAVE" button at the bottom of the card.
+Paste the IDP metadata descriptor into the bottom card
```

Comment:
Another small thing -- we're sort of using a mix of "IdP" and "IDP". Should probably just pick one and stick with it.

Reply:
Noted, will do. (Just use whatever we seem to use more often.)